src/ucp/core/ucp_ep.c

```diff
@@ -136,7 +136,8 @@ ucs_status_t ucp_ep_new(ucp_worker_h worker, const char *peer_name,
         goto err_free_ep;
     }
 
-    ucs_list_add_tail(&worker->all_eps, &ucp_ep_ext_gen(ep)->ep_list);
+    ucs_list_head_init(&ucp_ep_ext_gen(ep)->ep_list);
```

Comment:
seems redundant

Reply:
without this init it crashes when we do disconnect of this tmp ep in wireup ep destructor

Reply:
ok, thanks